wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -151,6 +152,16 @@ open class LockScreenActivity : SecureActivity() {
         }
     }
 
+    protected open fun turnOffAutoLogout() {
+        autoLogout.stopTimer()
+    }
+
+    protected open fun turnOnAutoLogout() {
```

Comment:
We have these methods in the `InteractionAwareActivity` for turning off autologout for liquid/uphold custom tabs. In crowdnode, it's needed for turning it off in the login webview.

Perhaps `LockScreenActivity` itself should derive from  `InteractionAwareActivity` so that we don't duplicate this code, however, the latest has some logic that finishes the activity on lock screen which we don't need here.

Reply:
Perhaps it should.